features/step_definitions/borrower_views_deed.rb

```diff
@@ -0,0 +1,30 @@
+
+Given(/^I have created a deed with two borrowers$/) do
+  step %(I have a valid deed)
+  step %(I create a deed using Deed API)
+  url = @response.body.to_s
+  split_url = url.split('/')
+  @deed_id = split_url[4]
+end
+
+Given(/^I navigate to the borrower frontend "([^"]*)" page$/) do |path|
+  visit(Env.borrower_frontend + path)
+end
```

Comment:
Think the url here might not be correct, as I read it, it going to http://10.10.10.10:9030/searchdeed?/deed/search. Might want to double check this, see my comment below for the env.rb file about using urls.